components/bsx/Offer/OffersUserTable.vue

```diff
@@ -88,6 +89,13 @@ export default class OffersUserTable extends mixins(
     return formatDistanceToNow(new Date(date), { addSuffix: true })
   }
 
+  get displayOffers() {
```

Comment:
I see that we use this getter in 2 other files, would it be possible to add it in `offermixin` or somewhere else to avoid repetition?

Reply:
make sense!